ConfigurationSystem/private/ConfigurationClient.py

```diff
@@ -17,30 +17,75 @@
 class ConfigurationClient(object):
 
   def __init__(self, fileToLoadList=None):
+    """ C'or
+
+        :param list fileToLoadList: files to load
+    """
     self.diracConfigFilePath = os.path.join(DIRAC.rootPath, "etc", "dirac.cfg")
     if fileToLoadList and isinstance(fileToLoadList, list):
       for fileName in fileToLoadList:
         gConfigurationData.loadFile(fileName)
 
   def loadFile(self, fileName):
+    """ Load file
+
+        :param basestring fileName: file name
```

Comment:
`basestring` is python 2 only, in 3 is simply `str`. For all these comments, should we put already `str` instead of `basestring`?

Reply:
yes, would make sense.